Editor/Window/Components/InfoLinks.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using AmazonGameLift.Editor;
+using Editor.Resources.EditorWindow;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace GameLiftPlugin.UI.Components
```

Comment:
These namespaces are kinda all over the place now. As far as I can tell, the previous iteration of the plugin used `AmazonGameLift.Editor` for all the editor code, then we started adding `Editor.filepath` namespaces and now `GameLiftPlugin.UI.Components` namespace. We should pick one structure and stick to it

Reply:
Do you have a preference?

Reply:
I'd prefer stick with `AmazonGameLift.Editor`. We can do a follow-up PR to clean up what was probably added before. But as we add new files we should stick to the one namespace

Reply:
Stuck to `AmazonGameLift.Editor`

Reply:
Oops! fixed now!